tools/get-browser-compat.js

```diff
@@ -0,0 +1,24 @@
+import z from "zod";
+
+import server from "../server.js";
+
+server.registerTool(
+  "get-browser-compat",
```

Comment:
Maybe keep it short?

```suggestion
  "get-compat",
```